sedlex.opam

```diff
@@ -5,9 +5,13 @@ maintainer: "Alain Frisch <alain.frisch@lexifi.com>"
 authors: [
   "Alain Frisch <alain.frisch@lexifi.com>"
 ]
-build: [ [make "all"]  [make "opt"] ]
-install: [make "install"]
-remove: [["ocamlfind" "remove" "sedlex"]]
+
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-doc: ["jbuilder" "build" "@doc" "-p" name "-j" jobs]
+build-test: ["jbuilder" "runtest" "-p" name "-j" jobs]
+install: ["jbuilder" "install" "-p" name "-j" jobs]
+remove: ["jbuilder" "uninstall" "-p" name "-j" jobs ]
```

Comment:
There's no need for `install:` or `remove:` here. This is because jbuilder will generate a .install file which is enough for opam to know how to do installation/uninstallation.

Reply:
Thanks. Fixed.